main.py

```diff
@@ -95,3 +97,12 @@
 
     # Print end timestamp
     print(f'>>>>>>>> End run {datetime.now()} <<<<<<<<<<')
+
+    # returns an exit code of either 0 (successful validation), or 1 (validation failed).
+    #
+    # If validation was not enabled, exit with a status of 0 (no errors)
+    
+    if validation_enabled:
+        exit(build_status)
+    else:
```

Comment:
I added an exit condition if validation was not enabled.

Reply:
Great! Thanks